From 54014061be2fed20d6c35aba9719ea70a9fea9ea Mon Sep 17 00:00:00 2001
From: Vojtech Trefny <vtrefny@redhat.com>
Date: Wed, 1 Sep 2021 08:59:19 +0200
Subject: [PATCH] Use setuptools instead of distutils in setup.py

The distutils module is deprecated and will be removed in Python
3.12.

Upstream-Status: Backport [https://github.com/kraj/blivet-gui/commit/54014061be2fed20d6c35aba9719ea70a9fea9ea]
---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index aa8ef57..77849a1 100644
--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,7 @@
 from __future__ import print_function
 
-from distutils.core import setup
-from distutils.command.sdist import sdist
+from setuptools import setup
+from setuptools.command.sdist import sdist
 import glob
 import sys
 
-- 
2.35.1

